.github/workflows/unit-tests-workflow-for-customImportMap.yml

```diff
@@ -0,0 +1,59 @@
+name: Unit tests workflow
+on:
+  push:
+    branches:
+      - "*"
+  pull_request:
+    branches:
+      - main
+
+env:
+  OPENSEARCH_DASHBOARDS_VERSION: '2.0'
+jobs:
+  tests:
+    name: Run unit tests
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Plugin
+        uses: actions/checkout@v1
```

Comment:
Why are you using v1 here? you are already using v2 in the next step

Reply:
my bad, will update it.